File: language/benchmarks/tokens/cases/topology-http-client/topology-http-client.ts
import { emptyHeaders, get } from "./http-client";
import { mailerApi } from "./topology";

export async function main():Promise<string>{
  const result=await get(mailerApi,emptyHeaders(),"/health");
  return result.ok ? `${result.value.status}:${result.value.body}` : `ERR:${result.error.message}`;
}

File: language/benchmarks/tokens/cases/topology-http-client/topology-http-client.py
from http_client import empty_headers, get
from topology import mailer_api


def main() -> str:
    result = get(mailer_api, empty_headers(), "/health")
    return f"{result['value']['status']}:{result['value']['body']}" if result["ok"] else f"ERR:{result['error']['message']}"
